python/lsst/obs/base/gen2to3/__init__.py

```diff
@@ -19,7 +19,16 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from .translators import *
+from .translators import (
+    AbstractToPhysicalFilterKeyHandler,
+    CalibKeyHandler,
```

Comment:
Please add a comment here about why this is different. We rarely do this, so the _why_ about this specific list is non-obvious.

Reply:
I was on the fence about this, and I'm now thinking the fact that it's so unusual makes it not worth doing, especially not to hide one symbol.  If it was common practice either in obs_base or in the stack at large to carefully control what we export, it'd make more sense (and we'd need to either use leading underscores or `__all__` in `__init__.py` a lot more, as we do in daf_butler).  But as it is, it's just not a good tradeoff.  I think I've going to revert this change.